src/inverseRobot.js

```diff
@@ -1,13 +1,29 @@
-'use strict';
+/* eslint-disable quotes */
+"use strict";
 
 /*
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
   // write code here
+  const res = {};
+  const valuesArr = Object.values(robot);
+  const isValuesRepeated = valuesArr.some(
+    (item, index) => valuesArr.indexOf(item) !== index
+  );
+
+  if (isValuesRepeated) {
+    return null;
+  }
+
+  for (const key in robot) {
+    res[robot[key]] = key;
+  }
```

Comment:
No need to do extra loops iterations. You can check it in your for loop
```suggestion
  for (const key in robot) {
    const newKey = robot[key];
    
    if (res.hasOwnProperty(newKey )) {
      return null;
    }
     
    res[newKey] = key;
  }
```